### ISSUE TITLE
Crash downloading subtitles

### ISSUE THREAD

**Issue by carl689** [Jsakkos/mkv-episode-matcher#4]
Trying this out for the first time and getting the following crash

```
2024-08-08 23:15:29.632 | INFO     | mkv_episode_matcher.utils:get_subtitles:163 - Found 20 episodes in Season 1
2024-08-08 23:15:29.636 | INFO     | mkv_episode_matcher.utils:get_subtitles:166 - Processing Season 1, Episode 1...
2024-08-08 23:15:29.799 | WARNING  | mkv_episode_matcher.utils:get_subtitles:184 - No subtitles found for Stargate Atlantis - S01E01
2024-08-08 23:15:29.804 | INFO     | mkv_episode_matcher.utils:get_subtitles:166 - Processing Season 1, Episode 2...
2024-08-08 23:15:29.963 | INFO     | mkv_episode_matcher.utils:get_subtitles:195 - Original filename: Stargate Atlantis S01E01 . S01E02 - Rising
2024-08-08 23:15:30.370 | ERROR    | mkv_episode_matcher.__main__:main:175 - An error has been caught in function 'main', process 'MainProcess' (39648), thread 'MainThread' (41388):
Traceback (most recent call last):

  File "C:\Users\carl6\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 815, in move
    os.rename(src, real_dst)
    │  │      │    └ 'C:\\Users\\carl6\\.mkv-episode-matcher\\cache\\data\\Stargate Atlantis\\Stargate Atlantis - S01E02.srt'
    │  │      └ '7796524.srt'
    │  └ <built-in function rename>
    └ <module 'os' from 'C:\\Users\\carl6\\AppData\\Local\\Programs\\Python\\Python310\\lib\\os.py'>

FileNotFoundError: [WinError 3] The system cannot find the path specified: '7796524.srt' -> 'C:\\Users\\carl6\\.mkv-episode-matcher\\cache\\data\\Stargate Atlantis\\Stargate Atlantis - S01E02.srt'
```

**Comment by Jsakkos** [comment]
Let me do some testing, as I recently ran into a similar issue. Do you have the subtitles already or are you trying to automatically download them? If the latter, did you get both Opensubtitles.com and TMDb?

**Comment by carl689** [comment]
Attempting to automatically download them.  Setup both Opensubtitles.com and TMDb

**Comment by Jsakkos** [comment]
I can reproduce the error, but I'll need some time to fix it. There's something about that specific file that is causing an issue. In the meantime, you can always download the srt files manually and place them into the appropriate folder to perform the mkv matching.

**Comment by carl689** [comment]
I will give that a shot tonight, thanks! 

Two more comments
 * You should put up a donation/coffee/sponsor thing, this project seems like it will really scratch an itch for people
 * Have you tried having it read the first X minutes per file? I would GUESS you would get unique enough subtitles from the first 5min of a tv show. (Long enough to get past any intro)

**Comment by Jsakkos** [comment]
Putting up a donation link is a great idea. I built this my project for myself, but I figured it would be of interest to others. I'll have to play around with minimizing the amount of the original file that's processed. I think you're probably right that 5 mins would be enough. The most unreliable part of the code is getting accurate subtitles from opensubtitles.com.

**Comment by Jsakkos** [comment]
This should be resolved in v0.1.6. If you still run into issues, try running the get_subtitles_test notebook that I added.
